rdr_service/model/bq_questionnaires.py

```diff
@@ -910,6 +910,80 @@ class BQPDRSDOHView(BQModuleView):
     _show_created = True
 
 
+class BQPDRCOPEVaccine3Schema(_BQModuleSchema):
+    """ COPE Vaccine Survey 3 (Winter 2021) """
+    _module = 'cope_vaccine3'
+    _force_boolean_fields = (
```

Comment:
Sheesh...

Reply:
There are some shared "forced boolean" fields between all the surveys that included vaccine questions (Feb Cope + the minute surveys?)

But I don't understand why this survey has questions about so many doses.   (Haven't seen the codebook)